### ISSUE TITLE
Question about the slip system 

### ISSUE THREAD

**Issue by supersrh** [dcelisgarza/DDD.jl#16]
I have little knowledge about DDD but want to learn and use it for material study.
I have a question about the "bVec" in the file "FCC.json"
The burgers vector for the {111}<110> slip system of FCC structure should be 1/2<110>, from (0, 0, 0) to (0.5, 0.5, 0) for instance. The JSON file gives <110>.
Could you perhaps offer more documentation on the theory and teach a rookie like me?

**Comment by supersrh** [comment]
I'm gonna to read your PHD thesis!

**Comment by dcelisgarza** [comment]
You're right, it's just that the factor usually gets offloaded to the mobility law.

I don't know if i'll continue with this project, my plan was to propose it as a post doc but that seems to have fallen through. It was my attempt at convincing people that there definitely is a better way of doing things.

In my phd I used and contributed to https://github.com/TarletonGroup/EasyDD which is more complete, but written in matlab.
